src/main/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlight.java

```diff
@@ -79,7 +81,10 @@ public SnapshotCreateFlight(FlightMap inputParameters, Object applicationContext
     addStep(
         new AuthorizeBillingProfileUseStep(profileService, snapshotReq.getProfileId(), userReq));
 
-    // Get or create the project where the snapshot resources will be created
+    // Get a new google project from RBS and store it in the working map
+    addStep(new CreateSnapshotGetProjectStep(bufferService));
+
+    // Get or initialize the project where the snapshot resources will be created
     addStep(new CreateSnapshotGetOrCreateProjectStep(resourceService, firestoreRegion));
```

Comment:
I'm not totally following why we need both of these steps. Don't we initialize the project w/ RBS in the first step?

Reply:
Thanks for walking me through this!